walk-in-interview/src/main/webapp/common-functions.js

```diff
@@ -13,6 +13,8 @@ import {AppStrings} from './strings.en.js';
 
 const STRINGS = AppStrings['common'];
 const JOB_ID_PARAM = 'jobId';
+const DEFAULT_PAGE_SIZE = 20;
+const DEFAULT_PAGE_INDEX = 0;
```

Comment:
why is DEFAULT_PAGE_INDEX needed, shouldn't it always be 0 when you load a screen with a paginated list?